aws_sagemaker_studio/frameworks/keras_pipe_mode_horovod/keras_pipe_mode_horovod_cifar10.ipynb

```diff
@@ -602,9 +609,9 @@
  "metadata": {
   "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "Python 3 (TensorFlow CPU Optimized)",
+   "display_name": "Python 3",
```

Comment:
Not sure how much this matters in this example, but I think generally, for Keras, you'd want the TF Optimized kernel.

Reply:
The reason I chose the TF kernel over the data science one was because the code for generating the training data requires `tensorflow` to be installed in the environment. (though I don't think the display name really matters all that much)